### PR TITLE
CLI update 'find' to 'list'

### LINKED ISSUE #753: Update CLI commands from 'find' to 'list'
All the `influx` CLI's `find` subcommands have been updated to `list` and now how `ls` and `find` as aliases.

PR: https://github.com/influxdata/influxdb/pull/16805

### CLI docs
- [x] /v2.0/reference/cli/influx/auth/
- [x] /v2.0/reference/cli/influx/auth/find/
- [x] /v2.0/reference/cli/influx/bucket/
- [x] /v2.0/reference/cli/influx/bucket/find/
- [x] /v2.0/reference/cli/influx/org/
- [x] /v2.0/reference/cli/influx/org/find/
- [x] /v2.0/reference/cli/influx/secret/
- [x] /v2.0/reference/cli/influx/secret/find/
- [x] /v2.0/reference/cli/influx/task/
- [x] /v2.0/reference/cli/influx/task/find/
- [x] /v2.0/reference/cli/influx/task/log/
- [x] /v2.0/reference/cli/influx/task/log/find/
- [x] /v2.0/reference/cli/influx/task/run/
- [x] /v2.0/reference/cli/influx/task/run/find/
- [x] /v2.0/reference/cli/influx/user/
- [x] /v2.0/reference/cli/influx/user/find/

### Task docs
- [x] /v2.0/process-data/manage-tasks/run-task/
- [x] /v2.0/process-data/manage-tasks/task-run-history/
- [x] /v2.0/process-data/manage-tasks/view-tasks/

### Org docs
- [x] /v2.0/organizations/view-orgs/

### Bucket docs
- [x] /v2.0/organizations/buckets/view-buckets/

### Member docs
- [x] /v2.0/organizations/members/add-member/

### User docs
- [x] /v2.0/users/view-users/

### Secrets docs
- [x] /v2.0/security/secrets/manage-secrets/view/

### Token docs
- [x] /v2.0/security/tokens/view-tokens/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #753 

- Updates all CLI `find` commands to `list`
- General link cleanup

**Don't merge until https://github.com/influxdata/influxdb/pull/16805 is merged**

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/query-data/guides/monitor-states.md [hunk 1]
```diff
@@ -148,9 +148,9 @@ Detect state changes with the `monitor.stateChanges()` function. To use the `mon
 
     {{< nav-icon "alerts" >}}
 
-2. If you haven't already, [create a check](/v2.0/monitor-alert/checks/create/) that stores statuses (`CRIT`, `WARN`, `INFO`, `OK` or `ANY`) in the `_level` column. <!-- specify how to do this with monitor.check() function or in UI, with check threshold or deadman? 
+2. If you haven't already, [create a check](/v2.0/monitor-alert/checks/create/) that stores statuses (`CRIT`, `WARN`, `INFO`, `OK` or `ANY`) in the `_level` column. <!-- specify how to do this with monitor.check() function or in UI, with check threshold or deadman?
```
Comment (kelseiv) [inline content/v2.0/query-data/guides/monitor-states.md]: comment still a question?

File: content/v2.0/query-data/guides/monitor-states.md [hunk 2]
```diff
@@ -148,9 +148,9 @@ Detect state changes with the `monitor.stateChanges()` function. To use the `mon
...
 3. Import the InfluxDB `monitor` package.
-4. In your query, the specify the check. <!--can users specify a Flux query with the `monitoring` bucket and _level field without specifying the check? does importing the monitor package create the `monitoring` bucket? 
+4. In your query, the specify the check. <!--can users specify a Flux query with the `monitoring` bucket and _level field without specifying the check? does importing the monitor package create the `monitoring` bucket?
```
Comment (kelseiv) [inline content/v2.0/query-data/guides/monitor-states.md]: still a question?

File: content/v2.0/reference/cli/influx/user/list.md [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
-    name: influx user find
+    name: influx user list
     parent: influx user
 weight: 201
+aliases:
+  - /v2.0/reference/cli/influx/user/find
 ---
 
-The `influx user find` command lists and searches for users in InfluxDB.
+The `influx user list` command lists and searches for users in InfluxDB.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/user/list.md]: searches and lists?

File: content/v2.0/reference/glossary.md [hunk 4]
```diff
@@ -918,8 +918,9 @@ Related entries: [bin](#bin)
 
 ### step-plot
 
-In InfluxDB 1.x, a [step-plot graph](https://docs.influxdata.com/chronograf/v1.7/guides/visualization-types/#step-plot-graph) displays time series data in a staircase graph.
-In InfluxDB 2.0, generate a similar graph using the step interpolation option for [line graphs](https://v2.docs.influxdata.com/v2.0/visualize-data/visualization-types/graph/#options).
+A data visualization that displays time series data in a staircase graph.
+Generate a step-plot using the step interpolation option for
```
Comment (kelseiv) [inline content/v2.0/reference/glossary.md]: using the interpolation option?

File: content/v2.0/query-data/guides/monitor-states.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/query-data/guides/monitor-states.md]: @kelseiv I just updated and/or commented out broken links in this doc. This was commented in your original draft of this doc.

File: content/v2.0/query-data/guides/monitor-states.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/query-data/guides/monitor-states.md]: Same as above

**Pull request merged**